source/AI.cpp

```diff
@@ -1267,29 +1285,6 @@ void AI::FindTarget(Ship &ship) const
 		}
 	}
 
-	// Run away if your hostile target is not disabled and you are badly damaged.
-	// Player ships never stop targeting hostiles, while hostile mission NPCs will
-	// do so only if they are allowed to leave.
-	if(!isYours && target && target->GetGovernment()->IsEnemy(gov) && !isDisabled
-			&& (person.IsFleeing() || (ship.Health() < (RETREAT_HEALTH + .25 * person.IsCoward())
-			&& !person.IsHeroic() && !person.IsStaying() && !parentIsEnemy)))
```

Comment:
I don't think `parentIsEnemy` is necessary? Although I'm not 100% sure.

Reply:
Agreed but I dont think we should change it in this PR honestly

Reply:
Ah rip you need this for mission NPCs.